jpegenc/subblocks/quantizer/divider.py

```diff
@@ -0,0 +1,129 @@
+"""This module contains the HDL for
+    divider used for Quantiser"""
+
+from myhdl import always_seq, block, intbv, always_comb, Signal
+from .romr import romr
+
+
+def divider_ref(dividend, divisor):
```

Comment:
Lets keep these test sources in the same places for time being. I still need to write reference models for couple things. Once we make them we will have a total software model for reference.
(This is just a suggestion).